lib/iiif/v3/abstract_resource.rb

```diff
@@ -5,22 +5,37 @@ module V3
     class AbstractResource
       include IIIF::HashBehaviours
 
-      # Every subclass should override the following methods defining key value types, see Subclasses for how
+      # properties used by content resources only
+      CONTENT_RESOURCE_PROPERTIES = %w{ format height width duration }
+
+      # used by Collection, AnnotationCollection
+      PAGING_PROPERTIES = %w{ first last next prev total start_index }
```

Comment:
nothing seems to use these new `_PROPERTIES` keys, though there is suggested usage below... are these something that will be used in the future?  by consumers of the gem?  something used by some meta-programming magic that i'm not seeing/understanding?